src/org/jitsi/impl/neomedia/RTPConnectorOutputStream.java

```diff
@@ -215,6 +238,12 @@ static boolean logPacket(long numOfPacket)
     private boolean closed = false;
 
     /**
+     * The instance used to calculate the sending bitrate of this output stream.
+     */
+    private RateStatistics rateStatistics
+        = new RateStatistics(AVERAGE_BITRATE_WINDOW_MS);
```

Comment:
I usually go with final here. Don't know whether that makes any difference. I appear to remember reading that final is overused and fields are probably the most useful places to use it... or something like that. I may be wrong.